internal/handler/executor.go

```diff
@@ -105,6 +107,8 @@ func (e *Executor) execHookCommand(w io.Writer) error {
 	if err != nil {
 		return err
 	}
+	// retrieve timeout value
+	timeout := e.hook.Timeout * time.Second
```

Comment:
I think something wrong is going on here. with Timeout being of type `time.Duration` you should NOT multiply this by `time.Second`, as if I pass e.g. `10m` as duration (which is 10 * time.Minute) and then here you multiply this by `time.Second` you overflow the int64 type, which is a backing type for `time` units. Duration does not need to be manipulated at all, you should use it directly as is.

```suggestion
	timeout := e.hook.Timeout
```

Reply:
I'm not sure if we can pass anything except int64 nanoseconds to default time.Duration realisation (unless we are operating Hook.Timeout as a string and use ParseDuration then)

Reply:
I'm not sure if we can pass anything except int64 nanoseconds to default `time.Duration` realisation through the `webhook.conf` (unless we are operating `Hook.Timeout` as a string and use `ParseDuration` then, which might be a more convenient way to represent timeout)